CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 0.7.1
+
+* Add `runStmt` for executing statements in the `IO` monad and binding new names.
+
```

Comment:
Please don't mention a new release. We don't know if it will be 0.7.1 or 0.8.0, for example.

If you want to start the new changelog entry, call it "Upcoming release".